offchain-modules/src/packages/db/entity/EosLock.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  PrimaryColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  Index,
+} from 'typeorm';
+
+@Entity()
+export class EosLock {
+  @PrimaryColumn()
+  @Index()
+  txHash: string;
+
+  @Index()
+  @Column()
+  sender: string;
+
+  @Index()
+  @Column()
+  token: string;
+
+  @Column()
+  amount: string;
+
+  @Index()
+  @Column()
+  recipientLockscript: string;
```

Comment:
For EOS lock event, there is no `recipientLockscript` and `sudtExtraData`, I think it's better to save `memo` in EosLock, and the parsed fields in `CkbMint`.

Reply:
ok